### DISCUSSION TITLE
recovering latent GP in LMC MOGP

### DISCUSSION THREAD

**Discussion by michaelcao28** [cornellius-gp/gpytorch#2182] [Q&A]
Hi there,

I am wondering if it is possible to obtain the latent GP in an LMC MOGP model, like the model in [this tutorial](https://docs.gpytorch.ai/en/stable/examples/04_Variational_and_Approximate_GPs/SVGP_Multitask_GP_Regression.html). In this tutorial, there were 3 latent GPs. Is there a way to retrieve the latent GP? Any tips would be much appreciated!

Thanks!

**Comment by gpleiss** [comment]
At the moment, there's not really a good (non-hacky) way to do this. It would make sense for there to be an argument that could get passed into LMCVariationalStrategy.__call__. Something like:

```python
model(x, latent_functions=True)  # latent_functions gets passed into LMCVariationalStrategy.__call__
```

Please feel free to open up a [feature request](https://github.com/cornellius-gp/gpytorch/issues/new?assignees=&labels=enhancement&template=---feature-request.md&title=%5BFeature+Request%5D) for this, but I'm pretty backlogged so I probably won't get to it for a while...
